Formula/sk.rb

```diff
@@ -0,0 +1,43 @@
+class Sk < Formula
+  desc "Fuzzy Finder in rust!"
+  homepage "https://github.com/lotabout/skim"
+  url "https://github.com/lotabout/skim/archive/v0.6.6.tar.gz"
+  sha256 "5975b21a7c3792e910ee9439b41b417e92f2fc7bc92d033e2d4c6d6c48a469c8"
+  head "https://github.com/lotabout/skim.git"
+
+  depends_on "rust" => :build
+
+  def install
+    (buildpath/"src/github.com/lotabout").mkpath
+    ln_s buildpath, buildpath/"src/github.com/lotabout/skim"
+    system "cargo", "install", "--root", prefix, "--path", "."
+
+    pkgshare.install "install"
+    (pkgshare/"shell").install %w[bash zsh].map { |s| "shell/key-bindings.#{s}" }
+    (pkgshare/"shell").install %w[bash zsh].map { |s| "shell/completion.#{s}" }
+    (pkgshare/"plugin").install "plugin/skim.vim"
+    man1.install "man/man1/sk.1", "man/man1/sk-tmux.1"
+    bin.install "bin/sk-tmux"
+  end
+
+  def caveats; <<~EOS
+    To install useful keybindings and fuzzy completion:
+
+    For bash, add to your bashrc:
+      source #{opt_prefix}/share/skim/shell/completion.bash
+      source #{opt_prefix}/share/skim/shell/key-bindings.bash
+
+    For zsh, add to your zshrc:
+      source #{opt_prefix}/share/skim/shell/completion.zsh
+      source #{opt_prefix}/share/skim/shell/key-bindings.zsh
+
+    To use skim in Vim, add the following line to your .vimrc:
+      set rtp+=#{opt_prefix}/share/skim
+  EOS
+  end
+
+  test do
+    (testpath/"list").write %w[hello world].join($INPUT_RECORD_SEPARATOR)
```

Comment:
What does `$INPUT_RECORD_SEPARATOR` evaluate to?

Reply:
Both empty and `\n` would work. Anyway I've removed it.